packages/web3torrent/src/pages/download/Download.tsx

```diff
@@ -25,12 +27,15 @@ const Download: React.FC<RouteComponentProps> = () => {
         <>
           <FormButton
             name="download"
+            spinner={loading}
```

Comment:
Does it make sense for the prop names to be `shouldShowSpinner`, `isShowingSpinner` , `isSpinning` or `isLoading`
This in addition to `loading` being changed to `isLoading`?

Reply:
We could definitely improve the names of the properties. I'll log this in a separate issue.